parlai/nn/lr_scheduler.py

```diff
@@ -161,7 +161,9 @@ def add_cmdline_args(
             '--max-lr-steps',
```

Comment:
Instead of leaving this here, maybe we should just give another long option to the one in train model?

Reply:
ahh that's not a bad idea